lib/libutils/ext/include/asm.S

```diff
@@ -34,6 +34,19 @@ UNWIND(	.fnstart)
 	BTI
 	.endm
 
+	.macro FUNC_RAW name colon section=default align=4
```

Comment:
Rather than different Macro's should I add naked as an apptibute to the FUNC, LOCAL_FUNC and WEAK_FUNC macro's ? This would avoid duplication also.
-       .macro FUNC name colon section=default align=4
+       .macro FUNC name colon section=default align=4 attribute=default
        .ifc    \section\(),default
        .section .text.\name
        .else
@@ -31,7 +34,10 @@
        .balign \align
        \name \colon
 UNWIND(        .fnstart)
+       .ifc    \attribute\(),naked
+       .else
        BTI
+       .endif
        .endm

Reply:
Good idea.